OIPA/.gitignore

```diff
@@ -1 +1,2 @@
-env
\ No newline at end of file
+env
+.env
```

Comment:
You want to ignore `.env` files to ignore virtual environment folders ? Unfortunately we can not do that, because settings in new OIPA are using https://github.com/theskumar/python-dotenv and its settings file is called `.env`. I think we will need to rename our virtual environments to `env` and `.env` files shouldn't be ignored.